### ISSUE TITLE
Hoisting static assets

### ISSUE THREAD

**Issue by samcorcos** [erikras/redux-form#2626]
### Are you submitting a **bug report** or a **feature request**?

Bug report

### What is the current behavior?

As far as I can tell, `redux-form` is not hoisting static assets. 

### What is the expected behavior?

I would expect it to hoist static assets.

### What's your environment?

```
redux-form: 6.4.3
node: 7.5.0
react-native: 0.41.0
redux: 3.6.0
react-redux: 4.4.6
```

### Other informations

I first thought this was a problem with `react-navigation` and I opened an issue ticket there:

https://github.com/react-community/react-navigation/issues/332#event-961713900

They said that it was an issue with `redux-form`, so I'm posting it here. I'm not sure how to diagnose this problem, but I'm happy to submit a PR if given some guidance.

**Comment by gustavohenke** [comment]
Yup.
`reduxForm` creates the following hierarchy of components:
`ReduxForm` -> `ConnectedForm` -> `Form` -> `YourWrappedForm`

The only place where hoisting happens is from `Form` to `ConnectedForm` - why?

**Comment by Kerumen** [comment]
ping @erikras

**Comment by edward-s** [comment]
im having the same issue.
any fixes?

**Comment by gustavohenke** [comment]
PRs are welcome.

**Comment by valerybugakov** [comment]
Bumped into the same issue ^

**Comment by erikras** [comment]
Published fix in [`v7.1.0`](https://github.com/erikras/redux-form/releases/tag/v7.1.0).
